### PR TITLE
remove loop in savemphtxt

### PR DESCRIPTION
**Pull request by brainwatcher**
as title discription. The fprintf function in for large loop takes too long time.

### CODE REVIEW

**Comment by fangq** [comment]
thanks for the patch - unfortunately I can't accept this one. the changes must be local and minimal, you should also not to remove the help info and the original author info.

if you are interested in revising this patch, please restore most of the changes (including white spaces), and keep the modifications to the least number of lines, and ensure no change to the functionality.

**Comment by brainwatcher** [comment]
I had added the author info back and committed again.

File: savemphtxt.m [hunk 1]
```diff
@@ -1,5 +1,4 @@
 function savemphtxt(node, face, elem, filename)
-%
```
Comment (fangq) [inline savemphtxt.m:2]: can you restore this empty line? without it, matlab has different behaviors

File: savemphtxt.m [hunk 2]
```diff
@@ -19,18 +18,17 @@ function savemphtxt(node, face, elem, filename)
 n_node = size(node,1);
 n_face = size(face,1);
 n_elem = size(elem,1);
-
-elem(:,1:4)=meshreorient(node(:,1:3),elem(:,1:4));
```
Comment (fangq) [inline savemphtxt.m:23]: I see you deleted this line as well as changing the below if/elseif conditions, are you sure these are equivalent?

File: savemphtxt.m [hunk 3]
```diff
@@ -40,36 +38,38 @@ function savemphtxt(node, face, elem, filename)
...
+
 fprintf(fp, '\n2\n\n3 tri\n');
 
 % Write Tri information
 fprintf(fp, '\n3\n');
 fprintf(fp, '%d\n\n',n_face);
-for i = 1:n_face
-    fprintf(fp,'%d %d %d\n',face(i,1),face(i,2),face(i,3));
-end
+% for i = 1:n_face
```
Comment (fangq) [inline savemphtxt.m:49]: please remove these commented lines if they are no longer needed. git keeps track of all edit history, so no need to keep those in the code.

**Comment by brainwatcher** [comment]
I missed the meshreorient line because my iso2mesh version is rather old. And I had rearranged my file. How about the third committed file?

**Pull request closed without merging**